### PR TITLE
Use GitHub issue templates

### PR DESCRIPTION
**Pull request by superkhau**
First run of using issue templates, please note we will should update this over time with the feedback we receive from community.

Connect to #2442 

@bajtos PTAL

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+**DO NOT ask questions using GitHub issues (we reserve GitHub issues for bug or feature requests only)**
```
Comment (Amir-61) [inline ISSUE_TEMPLATE.md]: for bug**s** or feature requests only

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+**DO NOT ask questions using GitHub issues (only bug or feature requests please see http://loopback.io/doc/en/contrib/Reporting-issues.html#asking-questions)**
+**Please ask questions at https://groups.google.com/forum/#!forum/loopbackjs or https://gitter.im/strongloop/loopback**
+**Immediate support is available through our subscription plans, see https://strongloop.com/node-js/subscription-plans**
```
Comment (superkhau) [inline ISSUE_TEMPLATE.md]: @bajtos I'm thinking we should move this into CONTRIBUTOR.md so we don't clutter up the template (that and it shows up as a yellow bar in in the GH UI.

Comment (bajtos) [inline ISSUE_TEMPLATE.md]: Let's use markdown comments for this. That way users filling a new issue will see the message, but the rendered issue won't show it.

```markdown
<!--
DO NOT ask questions using GitHub issues (only bug or feature requests please see http://loopback.io/doc/en/contrib/Reporting-issues.html#asking-questions)**
Please ask questions at https://groups.google.com/forum/#!forum/loopbackjs or https://gitter.im/strongloop/loopback**
Immediate support is available through our subscription plans, see https://strongloop.com/node-js/subscription-plans
-->
```

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+
+### Expected result
+
+
+### Actual result (if bug)
+
+
+
+### Link to sample repo to reproduce issue (if bug)
```
Comment (bajtos) [inline ISSUE_TEMPLATE.md]: Let's move this more to the top, before "expected" result.

In my experience, a good bug report has the following form:

 - steps to reproduce
 - actual result
 - expected result
 - additional information like Node/LoopBack versions, etc.

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+- Immediate support is available through our subscription plans, see
+  https://strongloop.com/node-js/subscription-plans
+-->
+
+### Bug or feature request
+
+- [ ] Bug
+- [ ] Feature request
+
+### Description of feature (or steps to reproduce if bug)
```
Comment (superkhau) [inline .github/ISSUE_TEMPLATE.md]: @bajtos Added steps to reproduce in this step and left description to cover the case where it is a feature (and therefore has no STR).

File: .github/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+
+
+### Link to sample repo to reproduce issue (if bug)
```
Comment (superkhau) [inline .github/ISSUE_TEMPLATE.md]: @bajtos Also moved this up above expected results as requested.

File: .github/ISSUE_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+### Expected result
+
+
+
+### Actual result (if bug)
+
+
+
+### Additional information (Node.js version, LoopBack version, etc)
```
Comment (superkhau) [inline .github/ISSUE_TEMPLATE.md]: @bajtos Grouped specific information request as requested.

**Comment by superkhau** [comment]
@bajtos PTAL again

**Comment by superkhau** [comment]
@rashmihunt soap connector is failing

**Comment by bajtos** [comment]
> @rashmihunt soap connector is failing

FWIW, we have a story (https://github.com/strongloop/loopback-connector-soap/issues/36) planned for this sprint to make the tests port-independent, although I don't know if that will fix the CI failure observed here.

**Comment by superkhau** [comment]
>Are we going to add pull request template too?

Yes, in another PR. ;)

**Comment by superkhau** [comment]
PR for PR template at https://github.com/strongloop/loopback/pull/2843

**Pull request merged**